DataFormats/EcalDigi/src/classes_def.xml

```diff
@@ -11,21 +11,30 @@
    <class name="EcalDataFrame" ClassVersion="10">
     <version ClassVersion="10" checksum="259364627"/>
    </class>
+   <class name="EcalDataFrame_Ph2" ClassVersion="-1">
```

Comment:
What is a ClassVersion -1?
I see another example in DataFormats/ForwardDetId/src/classes_def.xml , but honestly I don't know what it is intended for

Reply:
Dictionaries for new classes should be added with `ClassVersion="3"` ([SWGuideCreatingNewProducts#Class_versioning](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Class_versioning))

Reply:
Fixed